x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/datafeeds/RestPreviewDatafeedAction.java

```diff
@@ -35,8 +39,10 @@ public String getName() {
 
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) throws IOException {
-        String datafeedId = restRequest.param(DatafeedConfig.ID.getPreferredName());
-        PreviewDatafeedAction.Request request = new PreviewDatafeedAction.Request(datafeedId);
+        String datafeedId = restRequest.param(DatafeedConfig.ID.getPreferredName(), null);
```

Comment:
The `defaultValue` of `null` is unnecessary but I prefer this usage it makes the intent more explicit

Reply:
Gotcha, I will only grab this param if `hasContent` is false. Also, I won't provide a default value that way it blows up expecting it.